src/Estimators/MomentumDistribution.h

```diff
@@ -0,0 +1,117 @@
+//////////////////////////////////////////////////////////////////////////////////////
+// This file is distributed under the University of Illinois/NCSA Open Source License.
+// See LICENSE file in top directory for details.
+//
+// Copyright (c) 2020 QMCPACK developers.
+//
+// File developed by: Jaron T. Krogel, krogeljt@ornl.gov, Oak Ridge National Laboratory
+//
+// File refactored from: MomentumEstimator.h
+//////////////////////////////////////////////////////////////////////////////////////
+
+#ifndef QMCPLUSPLUS_MOMENTUMDISTRIBUTION_H
+#define QMCPLUSPLUS_MOMENTUMDISTRIBUTION_H
+#include <vector>
+
+#include "Configuration.h"
+#include "OperatorEstBase.h"
+#include "Containers/OhmmsPETE/TinyVector.h"
+
+#include "MomentumDistributionInput.h"
+
+
+namespace qmcplusplus
+{
+/** Class that collects momentum distribution of electrons
+ *  
+ */
+class MomentumDistribution : public OperatorEstBase
+{
+public:
+  using LatticeType = PtclOnLatticeTraits::ParticleLayout_t;
+  using RealType    = QMCTraits::RealType;
+  using ComplexType = QMCTraits::ComplexType;
+  using ValueType   = QMCTraits::ValueType;
+  using PosType     = QMCTraits::PosType;
+
+  //data members set only during construction
```

Comment:
Then all these members needs to be marked const. Probably myRNG is an exception.

Reply:
I removed myRNG (unused).  I've marked const where possible.  Some are set with computation within the constructor.